apps/frontend/src/pages/[type]/[id].vue

```diff
@@ -1664,6 +1664,12 @@ if (!route.name.startsWith('type-id-settings')) {
 const onUserCollectProject = useClientTry(userCollectProject)
 
 const { version, loader } = route.query
+if (
+	project.value.game_versions.length > 0 &&
+	project.value.game_versions.every((v) => v.includes('w') || v.includes('-'))
```

Comment:
this isn't how this should be checked, the release type should be checked in the tags list.

also, it doesn't seem like this hides the check box or anything?